docs/src/pages/Conformance.md

```diff
@@ -474,6 +474,7 @@ System operations are invoked at `[base]/$[operation]`
 | [$export](https://hl7.org/fhir/uv/bulkdata/STU1/OperationDefinition-export.html) | Export data from the server | exports to an S3-compatible data store; see the [user guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide#4101-bulk-data-export) for config info |
 | [$import](https://github.com/smart-on-fhir/bulk-import/blob/main/import.md) | Import FHIR Resources from a source| see the [user guide](https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide#4101-bulk-data-export) for config info. This implementation is based on the proposed operation.|
 | [$healthcheck](https://github.com/IBM/FHIR/blob/main/operation/fhir-operation-healthcheck/src/main/resources/healthcheck.json) | Check the health of the server | Checks for a valid connection to the database |
+| [$versions](https://hl7.org/fhir/capabilitystatement-operation-versions.html) | Returns the list of supported FHIR versions | Returning output in non-FHIR formats is not supported |
```

Comment:
> Returning output in non-FHIR formats is not supported
there actually is a way to coerce our operation framework to do this, but its not real pretty.  paul introduced it for the bulkdata-status operation

Reply:
Did you want that implemented right now in this issue?  Or can that wait?

Reply:
not sure what the $versions response is supposed to be...was just commenting on the statement in the doc

Reply:
ok, i looked it up:
> For the convenience of clients, servers that implement this operation SHALL also support the relevant XML or JSON format

so it does look like its required

Reply:
Correct, that's why for now I added the statement in the Conformance doc to say we don't support returning the output -- at least not yet.  If that needs to implemented in this PR, I'll need help to be pointed to what you mentioned that Paul did.  Otherwise, I can open a follow-on issue to add that support later, and we can prioritize it appropriately.

Reply:
I think I found Paul's example.  I'm going to give it a try.